tests/suggestion/index.js

```diff
@@ -0,0 +1,217 @@
+const request = require("supertest");
+const { grantPriviledge } = require("../helpers/strapi");
+
+describe("Suggestions controller", () => {
+  beforeAll(async () => {
+    // Grant priviledges to to controllers so they can be invoked anonymously
+    await Promise.all([
+      await grantPriviledge(
+        2,
+        "permissions.application.controllers.suggestion.new"
+      ),
+      await grantPriviledge(
+        2,
+        "permissions.application.controllers.suggestion.like"
+      ),
+      await grantPriviledge(
+        2,
+        "permissions.application.controllers.suggestion.unlike"
+      ),
+      await grantPriviledge(
+        2,
+        "permissions.application.controllers.suggestion.comment"
+      ),
+    ]);
+  });
+
+  describe("/suggestions/new", () => {
```

Comment:
Testaako tää koko end-pointin toimintaa vai vaan sitä, että end-point palauttaa arvot jotka se kuvittelee esim. menneen databaseen? PItäisikö tässä toi data validoida vielä data-basesta?

Reply:
Well this assumes that the values are in db also because the method returns the created entry.

But I guess it would be good to check the db also, I'll check if that's easily done.

Reply:
I updated the tests to verify updates from db also